src/Images.jsx

```diff
@@ -127,15 +127,16 @@ class Images extends React.Component {
 
         const runImage = (
             <Button key={image.Id + "create"}
+                    className="ct-container-create"
                     variant='secondary'
                     onClick={ e => {
                         e.stopPropagation();
                         this.setState({ showRunImageModal: image });
                     } }
-                    aria-label={_("Run image")}
                     isSmall
-                    icon={<PlayIcon />}
-                    data-image={image.Id} />
+                    data-image={image.Id}>
+                {_("Create container")}
```

Comment:
I think we need to rename this. `podman create` is not the same as `podman run`. We do the latter in fact.

If I recall correctly, we picked this name but then agreed there will be checkbox in modal 'Immediately start' or so. I don't see that in design. But until that is implemented, I think we need to name this 'Run'.

I'll add comment on the design and also add commit here.

Reply:
Makes sense!